upgrading/downgrade.adoc

```diff
@@ -176,9 +158,11 @@ include::day_two_guide/topics/proc_adding-etcd-after-restoring.adoc[leveloffset=
 include::day_two_guide/topics/proc_bringing-openshift-online.adoc[leveloffset=+1]
```

Comment:
This section need to be updated and confirmed with dev due to many config files are not avaiablable now. I'm not sure if it's enough.

Reply:
Etcd service is not running as systemd service in v3.10. So when downgrade to v3.10, we can not use original way to unmask/stop/start way to control etcd. Here need to be re-wroten.

Reply:
> # rpm -q openvswitch
Openvswitch service is not run as a systemd service. No here no need to verify it.

Reply:
Yes, I have an urgent request out to the devs and dev leadership to provide the updated etcd restore information, and I will update this PR as soon as I have new information.

Reply:
> > rpm -q openvswitch
> 
> Openvswitch service is not run as a systemd service. No here no need to verify it.

LGTM now

Reply:
I'd say that we should remove the bits about restoring etcd. Then the content of day_two_guide/topics/proc_bringing-openshift-online.adoc needs to be updated as it still references restoring control plane rpms, which is only valid for 3.10 to 3.9 downgrades.